### PR TITLE
Planning section

### PR DESCRIPTION
**Pull request by contactsanthosh21**
- Fixed background image
- Moved strings to constants in offering
- Added planning section
- Added transition to project impact hover

### CODE REVIEW

File: src/sections/Planning/Planning.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PlanningCard from './PlanningCard';
+var planningOneImage = require("../../assets/planning-1.png");
```
Comment (ankitg32) [inline src/sections/Planning/Planning.jsx]: nit: move the image imports to constants. and map over the constants to render PlanningCards

File: src/sections/Planning/Planning.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import Heading from '../../components/Heading/Heading';
+
+const Planning = () => {
+  const {planningOne,planningTwo,planningThree,planningFour} = constants.planning;
+
+  return (
+    <div className='planning section'>
+       <Heading className='planning__heading'>How do we do it?</Heading>
+       <div className='planning-cards'>
+          <PlanningCard image={planningOneImage} title={planningOne.title} description={planningOne.description}
```
Comment (ankitg32) [inline src/sections/Planning/Planning.jsx]: same as above comment. can use map function if planning objects have all the data inside them

File: src/sections/Planning/PlanningCard.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./PlanningCard.scss";
+import PlanningCardHover from './PlanningCardHover';
+
+const PlanningCard = ({image,title,description, children}) => {
+  const [style, setStyle] = React.useState({display: 'block'});
```
Comment (ankitg32) [inline src/sections/Planning/PlanningCard.jsx]: redundant states. u can have one state that toggles between hover and block state. will also reduce the handlers from two to just one and will make it a bit cleaner

File: src/sections/Planning/Planning.jsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+const Planning = () => {
+  const {planningOne,planningTwo,planningThree,planningFour} = constants.planning;
+
+  return (
+    <div className='planning section'>
+       <Heading className='planning__heading'>How do we do it?</Heading>
+       <div className='planning-cards'>
+          <PlanningCard image={planningOneImage} title={planningOne.title} description={planningOne.description} 
+          children={planningOne.hoverChildren}
```
Comment (ankitg32) [inline src/sections/Planning/Planning.jsx]: this is considered as a bad pattern. children is a prop reserved by react and overwriting it is not a good approach. you might want to consider passing render props, or just pass the right child simply as child and not explicitly as 
`          children={planningOne.hoverChildren}
`

File: src/sections/Planning/PlanningCard.scss
Comment (ankitg32) [inline src/sections/Planning/PlanningCard.scss:1]: convert all pixels to rem

File: src/sections/Planning/PlanningCardHover.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './PlanningCardHover.scss';
+
+const PlanningCardHover = ({children,className,style,setStyle,setStyleHover}) => {
+  className = 'planning-card__hover ' + className;
```
Comment (ankitg32) [inline src/sections/Planning/PlanningCardHover.jsx:5]: why call it __hover when it is not always going to be hover?

File: src/sections/Planning/PlanningCard.jsx [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    <>
+      <div className='planning-card' style={style} onMouseEnter={e => {
+        setStyle({display: 'none'});
+        setStyleHover({display: 'block'});
+      }}>
+        <img src={image}/>
+        <h3 >{title}</h3>
+        <p>{description}</p>
+      </div>
+      <PlanningCardHover className="hover-content" style={styleHover} setStyle={setStyle} setStyleHover={setStyleHover}>
```
Comment (ankitg32) [inline src/sections/Planning/PlanningCard.jsx]: why we call it PlanningCardHover when this is not always going to be in hover state? hover is a state, do we want to name component in reference to a state name?

File: src/sections/Planning/PlanningCardHover.jsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+        borderRadius:"1rem",
+        padding: "2.5rem 1.5rem",
+        cursor:"pointer",
+        marginTop: "4.5rem",
+        zIndex: 20,
+        height:"20rem",
+        ...style
+    }}
+    onMouseLeave={e => {
+        setStyle({display: 'block'});
```
Comment (ankitg32) [inline src/sections/Planning/PlanningCardHover.jsx]: same as above. can be done in one handler

File: src/sections/ProjectsImpact/ProjectsImpact.scss [hunk 8]
```diff
@@ -7,6 +7,7 @@
         gap:3.5rem;
         margin-top: 5rem;
         justify-content: center;
+        transition: 0.3s;
         &:hover{
             background: $color-white;
             box-shadow: 0px 0px 34px 0px #F1DDE4;
```
Comment (ankitg32) [inline src/sections/ProjectsImpact/ProjectsImpact.scss]: change px to rem

File: src/utils/constants.js
Comment (ankitg32) [inline src/utils/constants.js:1]: can we import images here too?

File: src/sections/Planning/Planning.jsx (hunk 1 above)
Comment (contactsanthosh21) [reply src/sections/Planning/Planning.jsx]: Done

File: src/sections/Planning/Planning.jsx (hunk 2 above)
Comment (contactsanthosh21) [reply src/sections/Planning/Planning.jsx]: Done

File: src/sections/Planning/Planning.jsx (hunk 4 above)
Comment (contactsanthosh21) [reply src/sections/Planning/Planning.jsx]: Done

File: src/sections/Planning/PlanningCard.scss
Comment (contactsanthosh21) [reply src/sections/Planning/PlanningCard.scss:1]: Done

File: src/sections/ProjectsImpact/ProjectsImpact.scss (hunk 8 above)
Comment (contactsanthosh21) [reply src/sections/ProjectsImpact/ProjectsImpact.scss]: Done

File: src/utils/constants.js
Comment (contactsanthosh21) [reply src/utils/constants.js:1]: Done

File: src/sections/Planning/PlanningCard.jsx (hunk 3 above)
Comment (contactsanthosh21) [reply src/sections/Planning/PlanningCard.jsx]: Done

File: src/sections/Planning/PlanningCardHover.jsx (hunk 7 above)
Comment (contactsanthosh21) [reply src/sections/Planning/PlanningCardHover.jsx]: Done

File: src/sections/Planning/PlanningCard.jsx (hunk 6 above)
Comment (contactsanthosh21) [reply src/sections/Planning/PlanningCard.jsx]: We are showing this component only on hover(mouse enter into parent)

File: src/sections/Planning/PlanningCardHover.jsx (hunk 5 above)
Comment (contactsanthosh21) [reply src/sections/Planning/PlanningCardHover.jsx:5]: We are showing this component only on hover(mouse enter into parent)

File: src/sections/Planning/Planning.jsx (hunk 1 above)
Comment (contactsanthosh21) [reply src/sections/Planning/Planning.jsx]: Also added the map function for offering section too

**Review by ankitg32 — APPROVED** [review]

**Pull request merged**